CMakeLists.txt

```diff
@@ -302,7 +302,7 @@ function(strip_symbols targetName outputFilename)
           TARGET ${targetName}
           POST_BUILD
           VERBATIM 
-          COMMAND ${OBJCOPY} --only-keep-debug ${strip_source_file} ${strip_destination_file}
+          COMMAND ${OBJCOPY} ${strip_source_file} ${strip_destination_file}
```

Comment:
The .dbg files are not meant to be used as the executable as in the above example.  They are meant to be installed side-by-side with the.so file and the native debuggers (lldb and gdb) will automatically use the symbols from them.